.github/workflows/build_mpv_binary_and_deb_package.yml

```diff
@@ -0,0 +1,139 @@
+name: Build mpv binary and create debian package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            build-essential \
+            clang \
+            yasm \
+            git \
+            meson \
+            ninja-build \
+            autoconf \
+            automake \
+            libtool \
+            pkg-config \
+            libfreetype6-dev \
+            libfribidi-dev \
+            libfontconfig1-dev \
+            libasound2-dev \
+            pulseaudio \
+            libx11-dev \
+            libxss-dev \
+            libvdpau-dev \
+            libgl-dev \
+            libxv-dev \
+            libjpeg-dev \
+            libssl-dev \
+            libx264-dev \
+            libmp3lame-dev \
+            libfdk-aac-dev \
+            python3-pip \
+            python3.12-venv \
+            libharfbuzz-dev \
+            nasm \
+            liblua5.1-0-dev \
+            liblua5.2-dev \
+            libxext-dev \
+            libxrandr-dev \
+            libxinerama-dev \
+            libxv-dev \
+            libwayland-dev \
+            nodejs \
```

Comment:
nodejs is definitely not needed

Reply:
after running `./rebuild -j4` command, I get the following result. 

```
mpv 0.39.0-UNKNOWN

    d3d11     : NO
    javascript: YES
    libmpv    : NO
    lua       : YES
    opengl    : YES
    vulkan    : YES
    wayland   : YES
    x11       : YES
   ```

To turn `javascript` option to `YES`, I uploaded `nodejs`. But I can certainly delete if it is needed at all.

Reply:
> To turn `javascript` option to `YES`, I uploaded `nodejs`

mpv's js doesn't use nodejs. It uses mujs, which is a tiny interpreter.

Reply:
@avih thank you!